Packs/ApiModules/ReleaseNotes/2_2_22.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### TAXII2ApiModule
+Changed the condition used to determine, which version of TAXII 2 to use - 2.0 or 2.1.
```

Comment:
```suggestion
- Fixed implementation of determining the TAXII version to use.
```

Reply:
I'm fine with the modified sentence, but it should not start with -, as it is against PaloAlto rules for release notes.

Reply:
https://xsoar.pan.dev/docs/documentation/release-notes#md-formatting - for single line change it should be without "-"